packages/dashboard/src/components/Wizard/Wizard.types.ts

```diff
@@ -43,11 +45,15 @@ export interface IWizardStepProps {
 }
 
 export interface IWizardContentProps {
-  contentTitle?: string;
+  contentTitleElement?: JSX.Element;
 
   content: JSX.Element;
 
   mainAction: IWizardStepAction;
```

Comment:
Do we still keep the three actions now that action bar is provided as JSX.Element?

Reply:
I thought mainAction and exitWizardAction are required for all steps, and backaction is also common if implemented.  Hence I provided these for use.

If there are more actions, they can add.

Reply:
Should this be part of wizard property though? Wizard does not invoke the actions.

---
In reply to: [260062201](https://github.com/OfficeDev/office-ui-fabric-react/pull/8115#discussion_r260062201) [](ancestors = 260062201)

Reply:
actions should be specific to each step.  Anyway, I have removed them now.